nx/include/switch/services/hid.h

```diff
@@ -600,6 +600,18 @@ typedef struct HidVibrationValue
     float freq_high; ///< High Band frequency in Hz.
 } HidVibrationValue;
 
+static inline u32 hidControllerIDToOfficial(HidControllerID id) {
+    if (id < CONTROLLER_HANDHELD) return id;
+    if (id == CONTROLLER_HANDHELD) return 0x20;
+    return 0x10;//For CONTROLLER_UNKNOWN and invalid values return this.
+}
+
+static inline HidControllerID hidOfficialToControllerID(u64 id) {
```

Comment:
A better name could be `hidControllerIDFromOfficial`. Why is the id parameter 64-bit unlike the other func?